src/libraries/System.Collections/src/System/Collections/Generic/HashSet.cs

```diff
@@ -241,6 +252,9 @@ public void Clear()
                 // clear only up to _lastIndex for _slots
                 Array.Clear(_slots, 0, _lastIndex);
                 Array.Clear(_buckets, 0, _buckets.Length);
+#if BIT64
+                _fastModMultiplier = HashHelpers.GetFastModMultiplier(0);
```

Comment:
I don't think this is right -- the multiplier goes with the buckets array. When the buckets array is resized, it changes, otherwise it does not.

Reply:
Yes.  I'll get rid of this.  Was an early morning enhancement.